examples/generate.jl

```diff
@@ -31,3 +31,6 @@ set_cell!(frame, UnitCell(10, 10, 10))
 
 trajectory = Trajectory("water-co2.pdb", 'w')
 write(trajectory, frame)
+# When running on the REPL, remember to close the trajectory or else it won't
```

Comment:
I would put this in the doc of `close` too. 

Also, these files are included in  the tutorials (http://chemfiles.org/Chemfiles.jl/v0.8.0/tutorials.html), maybe we can add this comment there too/instead?